tests/composition/test_composition.py

```diff
@@ -3436,6 +3436,60 @@ def test_call_beforeafter_values_twopass(self):
 #         assert 16 == output[0][0]
 #
 
+
+class TestSchedulerConditions:
+    @pytest.mark.composition
+    @pytest.mark.parametrize("mode",['Python',
+                            #  pytest.param('LLVM', marks=pytest.mark.llvm), #FIXME: Fails for `LLVM` and `LLVMExec` modes?
+                            #  pytest.param('LLVMExec', marks=pytest.mark.llvm),
```

Comment:
`LLVM` is using python scheduler, so it's not going to work.
`LLVMExec` should work and be fixed before merging